tests/Integration/Behaviour/Features/Context/Domain/AddressFeatureContext.php

```diff
@@ -341,6 +344,27 @@ public function checkOrderAddress(string $orderReference, string $addressReferen
         );
     }
 
+    /**
+     * @Then product :productReference is added to a cart :cartReference for :customerReference
```

Comment:
thanx for the tests !

here it should probably be "When" instead of "Then" since this method doesn't check anything, it performs an update, if I'm not wrong ?

Reply:
you're right! I've just fixed a few other wrongly assigned tests in this file :)